lxd/instance/drivers/driver_lxc.go

```diff
@@ -352,7 +353,7 @@ func lxcCreate(s *state.State, args db.InstanceArgs, revert *revert.Reverter) (i
 	}
 
 	d.logger.Info("Created container", log.Ctx{"ephemeral": d.ephemeral})
-	d.lifecycle("created", nil)
+	d.lifecycle(lifecycle.CREATED, nil)
```

Comment:
As lifecycle actions will end up covering more than just instances, we definitely want something like `InstanceCreated` instead. That way we can properly handle `NetworkCreated`, `ProfileCreated`, ...